src/ui/handler/shim/touch_zoom_rotate.js

```diff
@@ -77,7 +77,7 @@ export default class TouchZoomRotateHandler {
     /**
      * Returns true if the handler is enabled and has detected the start of a zoom/rotate gesture.
      *
-     * @returns {boolean}
+     * @returns {boolean} `true` if enabled a zoom/rotate gesture was detected.
```

Comment:
```suggestion
     * @returns {boolean} `true` if enabled and a zoom/rotate gesture was detected.
```
^ Added "and" to connect the two requirements.